DasharoPayloadPkg/Library/CbParseLib/CbParseLib.c

```diff
@@ -991,3 +991,39 @@ ParseCapsules (
 
   return RETURN_SUCCESS;
 }
+
+/**
+  Parse firmware information passed in by coreboot
+
+  @param  Guid     Kind of the firmware.
+  @param  Version  Current version.
+  @param  Lsv      Lowest supported version.
+
+  @retval RETURN_INVALID_PARAMETER  At least one of the parameters is NULL.
+  @retval RETURN_SUCCESS            Successfully parsed capsules.
+  @retval RETURN_NOT_FOUND          coreboot table is missing.
+**/
+RETURN_STATUS
+EFIAPI
+ParseFwInfo (
+  OUT EFI_GUID  *Guid,
+  OUT UINT32    *Version,
+  OUT UINT32    *Lsv
+  )
+{
+  struct lb_efi_fw_info  *FwInfo;
+
+  if (Guid == NULL || Version == NULL || Lsv == NULL) {
+    return RETURN_INVALID_PARAMETER;
+  }
+
+  FwInfo = FindCbTag (CB_TAG_FW_INFO);

```

Comment:
At some point we will need something like `FindNextCbTag`, I believe it would be useful in some parts of existing code, but this isn't something to worry about just yet.